.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: hostedgraphite/pythonbuild:hgapp_dep_xenial
+    steps:
+      - checkout
+      - run: apt-get update
+      - run: apt-get install -y libpq-dev lm-sensors
+      - run: pip install -r .travis.requirements.txt
+      - run: pip install coverage
+      - run: make install
+      - run: coverage run --source=diamond test.py
+      - store_artifacts:
+          path: /tmp/artifacts
```

Comment:
This step is probably redundant
![image](https://user-images.githubusercontent.com/40685761/155095824-5b566ba3-e4b3-4f83-a4ee-75c5de0ce057.png)